README.md

```diff
@@ -26,6 +26,16 @@ The libraries should be sorted alphabetically to make it easy to find them. Wher
 
 Note that this project is public to allow it to be easily accessed from local and CI builds.
 
+# included licences
+
+Centralized list on licences included/allowed in Alfresco software.
+
+NOTE: licences that are going be overridden should be present in the file
+
+# licences merges
+
+Centralized list of alias for the same licenses 
```

Comment:
This documentation doesn't quite make sense to me. I'm happy to help rewrite it, but let's work out how to phrase it together.